docker-for-mac/index.md

```diff
@@ -331,14 +334,112 @@ This option removes/resets all Docker data _without_ a reset to factory defaults
 
 ![Uninstall or reset Docker Edge features](images/settings-uninstall-edge.png)
 
+## Adding TLS certificates
+
+You can add server side (registry) and client side certificates to verify the identity of these entities.
```

Comment:
Apologies, one more nitpick:  it's not clear what "these entities" refer to.  Maybe this sentence can be:

"You can add CA certificates (used to verify registry server certificates) and client certificates (used to authenticate to registries) to your docker daemon."

Or something like that?